test/ctr.bats

```diff
@@ -188,9 +188,9 @@ function check_oci_annotation() {
 	crictl rm "$ctr_id"
 
 	# priority of 5 is LOG_NOTICE
-	journalctl -t conmon -p info CONTAINER_ID_FULL="$ctr_id" | grep -F "here is some output"
+	journalctl -p info CONTAINER_ID_FULL="$ctr_id" | tee /tmp/out | grep -F "here is some output"
```

Comment:
Why do we need the `tee /tmp/out` here?

Reply:
oopsies, that was left over from debugging haha